push-athena-data-to-rds/athena/queries.go

```diff
@@ -277,6 +277,7 @@ func FrontendHaproxyDataQuery(queryParams map[string]string, db string, table st
 	whereClause := sq.And{sq.Eq{"elb_status_code": fmt.Sprint("'", "200", "'")},
 		sq.NotLike{"request_url": fmt.Sprint("'", "%/?uptime%", "'")},
 		sq.NotLike{"request_url": fmt.Sprint("'", "%robots.txt%", "'")},
+		sq.NotLike{"request_url": fmt.Sprint("'", "%/api/v1%", "'")},
```

Comment:
Any reason for adding `v1`? can we remove all `\api\` ?

Reply:
That would skip api/auth also. We still want to count BrKe requests

Reply:
Thanks for the clarification